scripts/radix-zone/radix_zone_c2.env

```
@@ -65,6 +65,7 @@ AZ_RESOURCE_ACR_TASK_NAME="radix-image-builder"
 AZ_RESOURCE_CONTAINER_REGISTRY="radix${RADIX_ZONE}${RADIX_ENVIRONMENT}" # Note - ACR names cannot contain "-" due to reasons... radixc2prod
 AZ_RESOURCE_LOG_ANALYTICS_WORKSPACE="radix-container-logs-${RADIX_ZONE}-${RADIX_ENVIRONMENT}"
 AZ_RESOURCE_DNS="${RADIX_ZONE}.radix.equinor.com"
+AZ_RESOURCE_STORAGEACCOUNT_FLOW_LOGS="radixflowlogs${RADIX_ZONE}"
```

`C2 prod` cluster includes `RADIX_ENVIRONMENT` as naming convention.
```suggestion
AZ_RESOURCE_STORAGEACCOUNT_FLOW_LOGS="radixflowlogs${RADIX_ZONE}${RADIX_ENVIRONMENT}"
```